.github/workflows/ci-vulnerability-scans.yml

```diff
@@ -0,0 +1,82 @@
+# This workflow will scan the release versions of your Docker images, and standardize the results into a JSON file. This file
+# can be used to create Github tickets and Slack alerts
+
+name: Template Release Image Scans
+
+on:
+  schedule:
+    - cron: '0 0 * * *'  # Run daily at midnight UTC
+  pull_request:
+    paths:
+      # Handles when there are changes to the ignore files
+      - .grype.yml        # Anchore ignore
+      - .hadolint.yaml    # Hadolint ignore
+      - .trivyignore.yaml # Trivy ignore
+      - trivy.yaml        # Trivy config that points to the ignore file
+      # Handles when there are changes to the actual workflows we use
+      - .github/workflows/ci-vulnerability-scans.yml
+      - .github/workflows/vulnerability-scans.yml
+      - .github/actions/image-scan/action.yml
+      # Add paths to monitor for changes that should trigger a scan
+      ## Set for this example repo
+      - app/Dockerfile
+      - app/requirements.txt
+      - app/.dockerignore
+      - app-rails/Dockerfile
+      - app-rails/Gemfile
+      - app-rails/Gemfile.lock
+      - app-rails/package.json
+      - app-rails/package-lock.json
+      - app-rails/.dockerignore
+    # So we can run this workflow manually
+    workflow_dispatch:
+
+# These permissions are needed for opening Github tickets, remove comments if that functionality is enabled
+# permissions:
+#     contents: read
+#     issues: write
+
+jobs:
+  matrix_image_scans:
+    # Setting to new in case there's anything checking the existing vulnerability-scans.yml
+    # This will be fixed before merging
+    uses: ./.github/workflows/new-vulnerability-scans.yml
+    strategy:
```

Comment:
This might be the issue from [here](https://github.com/navapbc/platform-test/actions/runs/13081157307). In my file I pulled this from, I have the matrix in the workflow this calls, but I forgot why I did that. I think this could be why I did it like that. If so, should be an easy move into the sub-workflow to test it